doc/ref_model/chapters/chapter07.rst

```diff
@@ -1143,14 +1141,11 @@ System hardening
 |                 | the source code and manifests.                   |                                                 |
 +-----------------+--------------------------------------------------+-------------------------------------------------+
 | req.sec.gen.010 | The cloud infrastructure **must** support        |                                                 |
```

Comment:
This should look like this: 
![image](https://github.com/user-attachments/assets/9c29423b-c5b8-4dd6-9256-a87aa0262fe9)

Reply:
Sorry my previous suggestion was not working for the linkcheck. 

This one works:
![image](https://github.com/user-attachments/assets/7fe66622-da7f-43da-a1cb-f0a0472e20cc)

Reply:
Solved